packages/react-components/react-components/config/api-extractor.unstable.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "extends": "@fluentui/scripts/api-extractor/api-extractor.common.v-next.json",
+  "mainEntryPointFilePath": "<projectFolder>/dist/types/unstable/index.d.ts",
```

Comment:
this is needed to process `unstable` api. thank to this we now have a common approach for `unstable` apis introduction to any package